src/commands/support/manualleg.ts

```diff
@@ -0,0 +1,19 @@
+import { CommandDefinition } from '../../lib/command';
+import { makeEmbed, makeLines } from '../../lib/embed';
+import { CommandCategory } from '../../constants';
+
+const MANUAL_URL = 'https://cdn.discordapp.com/attachments/752801628347957248/929071827752480839/mcdu-discontinuity-manual-conceptual.png';
+const DISCON_DOCS_URL = 'https://docs.flybywiresim.com/pilots-corner/advanced-guides/flight-planning/disco/?h=manual#discontinuities-after-manual-leg'
+
+export const manualleg: CommandDefinition = {
+    name: ['manualleg','vm'],
```

Comment:
```suggestion
    name: ['manualleg', 'vm'],
```

This was suggested before, but not done.